packages/api/endpoints/reconciliation-reports.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable no-param-reassign */
+
+'use strict';
+
+const _get = require('lodash.get');
+const path = require('path');
+const { inTestMode } = require('@cumulus/common/test-utils');
+const { aws, log } = require('@cumulus/common');
+const { invoke } = require('@cumulus/ingest/aws');
+const handle = require('../lib/response').handle;
+
+/**
+ * List all reconciliation reports
+ *
+ * @param {Object} event - aws lambda event object.
+ * @param {function} cb - aws lambda callback function
+ * @returns {Array} - list of reports
+ */
+function list(event, cb) {
+  const constructResultFunc = (fileNames) =>
+    ({
+      meta: {
+        name: 'cumulus-api',
+        stack: process.env.stackName
+      },
+      results: fileNames
+    });
+
+  const systemBucket = process.env.system_bucket;
+  const key = `${process.env.stackName}/reconciliation-reports/`;
+  return aws.listS3ObjectsV2({ Bucket: systemBucket, Prefix: key })
+    .then((fileList) =>
+      fileList.filter((s3Object) => !s3Object.Key.endsWith(key))
```

Comment:
What is this filtering out?

Reply:
listS3ObjectsV2 returns the prefix as well which is actually the folder.

Reply:
👍